### DISCUSSION TITLE
Cypress Component testing code-coverage error.

### DISCUSSION THREAD

**Discussion by yanan5** [cypress-io/cypress#20987] [Component Testing]
I am trying to set-up code-coverage for cypress component testing in react [create-react-app].
Node - v14.2.0, npm - 7.6.3 on macOS Monterey version 12.1
Following is the package.json dev-dependencies
<img width="467" alt="Screenshot 2022-04-08 at 10 21 58 AM" src="https://user-images.githubusercontent.com/5122755/162369014-a077f05e-ffd1-43c7-8b3d-3a71bbbff128.png">
cypress.json
<img width="648" alt="Screenshot 2022-04-08 at 10 22 54 AM" src="https://user-images.githubusercontent.com/5122755/162369025-3a064cf0-948e-4e5c-8946-cf64a116ff66.png">
commands
<img width="964" alt="Screenshot 2022-04-08 at 10 23 05 AM" src="https://user-images.githubusercontent.com/5122755/162369051-2f55477c-a6db-4426-8ccc-ecd919e16f12.png">
Plugins
<img width="1297" alt="Screenshot 2022-04-08 at 10 23 34 AM" src="https://user-images.githubusercontent.com/5122755/162369069-271d7c5d-9c2e-430d-95a4-062978a04ac7.png">
<img width="1294" alt="Screenshot 2022-04-08 at 10 23 52 AM" src="https://user-images.githubusercontent.com/5122755/162369079-e30bbed9-59db-4535-83cb-6266b92a331b.png">

I am getting the following error after adding code-coverage for Cypress Component testing.
Note: It happens for one of the spec file only. If I skip the spec, code coverage works fine.
Any help in debugging or fixing it would be helpful. Please find below the details with screenshot
<img width="352" alt="Screenshot 2022-04-08 at 10 25 20 AM" src="https://user-images.githubusercontent.com/5122755/162368704-1ab666b4-67ec-4241-b1ce-868c58f016d7.png">
which points to code instrumented by cypress.
<img width="1784" alt="Screenshot 2022-04-08 at 10 25 43 AM" src="https://user-images.githubusercontent.com/5122755/162368738-67994ca5-e26a-4f5f-93e4-c8d0f918e6fb.png">
<img width="1238" alt="Screenshot 2022-04-08 at 10 26 02 AM" src="https://user-images.githubusercontent.com/5122755/162368745-c3a070bc-fe0d-4615-8b65-fabe3f8b6a22.png">
Please let me know if you need more information.

**Comment by muratkeremozcan** [comment]
I believe code coverage for component runner is not working in version 9.

e2e runner works.

Here's the repro repo: https://github.com/muratkeremozcan/react-hooks-in-action-with-cypress

I am waiting for version 10 to try both.
